modules/agar/src/test/ts/atomic/api/TestStoreTest.ts

```diff
@@ -0,0 +1,126 @@
+import { describe, it } from '@ephox/bedrock-client';
+import { Arr, Fun } from '@ephox/katamari';
+
+import * as GeneralSteps from 'ephox/agar/api/GeneralSteps';
+import { Pipeline } from 'ephox/agar/api/Pipeline';
+import { Step } from 'ephox/agar/api/Step';
+import { TestStore } from 'ephox/agar/api/TestStore';
+
+interface TestData {
+  readonly name: string;
+  readonly age: number;
+  readonly isHappy: boolean;
+}
+
+describe('atomic.agar.api.TestStoreTest', () => {
+  it('TestStore strings', () => {
```

Comment:
Nice tests. Maybe spilt the Step tests and the non-Step tests into different `context` definitions, to make the separation a bit more obvious.